src/library/scala/Array.scala

```diff
@@ -55,6 +55,19 @@ object Array {
    */
   def newBuilder[T](implicit t: ClassTag[T]): ArrayBuilder[T] = ArrayBuilder.make[T](t)
 
+  /** Builds an array from the iterable collection
```

Comment:
From Javadoc convention, first line always ends in a "period". My note would be, "Build" an array, imperative. For a class or "object-oriented" component, it "builds" an array.